frontend/fingertips-frontend/components/forms/SearchForm/SearchForm.test.tsx

```diff
@@ -22,14 +23,33 @@ jest.mock('react', () => {
   };
 });
 
+const initialState: SearchFormState = {
+  indicator: '',
+  message: null,
+  errors: {},
+};
+
 test('snapshot test - renders the form', () => {
-  const container = render(<SearchForm indicator="" />);
+  const container = render(
+    registryWrapper(<SearchForm searchFormState={initialState} />)
+  );
 
   expect(container.asFragment()).toMatchSnapshot();
 });
 
 test('should have an input field to input the indicatorId', () => {
-  render(<SearchForm indicator="" />);
+  render(registryWrapper(<SearchForm searchFormState={initialState} />));
 
   expect(screen.getByTestId('search-form-input-indicator')).toBeInTheDocument();
 });
+test('should display the error summary component when there is a validation error', () => {
```

Comment:
very minor but can you add a blank line inbetween the tests please. There is a plugin we can add that enforces this which would be nice to add https://www.npmjs.com/package/eslint-plugin-jest-formatting 🥇